.gitignore

```diff
@@ -1,5 +1,6 @@
 .vscode/
 .idea/
+.ssh
```

Comment:
How is this new added folder related to the project or to the changes of this PR?

Reply:
This is related to the changes of this PR. I wanted to stage all the files without staging the .ssh file, so I thought I might add it to .gitignore so that it doesn't get tracked. Do you have an alternative solution?